docs/source/advanced.rst

```diff
@@ -6,19 +6,19 @@ Advanced Usage
 Save Tabs
 ---------
 
-Everything in OnionShare is temporary by default. If you close an OnionShare tab, its address no longer exists and it can't be used again.
-Sometimes you might want an OnionShare service to be persistent.
-This is useful if you want to host a website available from the same OnionShare address even if you reboot your computer.
+Everything in OnionShare is temporary by default. Addresses no longer exist and can't be used again when OnionShare tabs are closed.
+Your OnionShare service can also be persistent.
+If host a website, percistence means it will be available on the same OnionShare address even if you reboot your computer.
 
-To make any tab persistent, check the "Save this tab, and automatically open it when I open OnionShare" box before starting the server.
-When a tab is saved a purple pin icon appears to the left of its server status.
+To make any tab persistent, check the "Save this tab, and automatically open it when I open OnionShare" box before starting your server.
+A purple pin icon appears to the left of its server status to tell you the tab is saved.
 
 .. image:: _static/screenshots/advanced-save-tabs.png
 
-When you quit OnionShare and then open it again, your saved tabs will start opened.
-You'll have to manually start each service, but when you do they will start with the same OnionShare address and private key.
+When opening OnionShare, your saved tabs from the prior session will start opened.
+Each service then can be started manually, and will be available on the same OnionShare address and be protected by the same private key.
 
-If you save a tab, a copy of that tab's onion service secret key will be stored on your computer with your OnionShare settings.
+If you save a tab, a copy of its onion service secret key is stored on your computer with your OnionShare settings.
```

Comment:
"with your OnionShare settings" isn't very clear. In the settings, settings for (?) is also saved alongside the key?

Reply:
What this means is that OnionShare settings are all stored in the same folder on the disk (in different places depending on the user's OS), and it's letting them know that secret keys will be stored in that folder as well. Though it could just say, "f you save a tab, a copy of its onion service secret key is stored on your computer."

Reply:
"with your OnionShare settings" can just be removed.